tensorflow/core/platform/default/logging.h

```diff
@@ -40,6 +40,7 @@ class LogMessage : public std::basic_ostringstream<char> {
  public:
   LogMessage(const char* fname, int line, int severity);
   ~LogMessage();
+  static int64 MinVLogLevel();
```

Comment:
Can you add documentation about what this?

// Returns the minimum log level for VLOG statements.
//
// E.g., if MinVLogLevel() is 2, then VLOG(2) statements will produce output, 
// but VLOG(3) will not.  Defaults to 0.